config/config.go

```diff
@@ -140,6 +140,7 @@ type TracingConfig struct {
 
 // IstioConfig describes configuration used for istio links
 type IstioConfig struct {
+	IstioStatusEnabled     bool   `yaml:"istio_status_enabled,omitempty"`
```

Comment:
Note that this requires a change in the operator.
In the Kiali CR and in the defaults.

Reply:
I've just created this PR into the kiali-operator to support this one.
Do I have to change any other place for the default? I've added one default at Line 329 at this same file.